combo_cases_and_deaths/delphi_combo_cases_and_deaths/run.py

```diff
@@ -77,18 +117,13 @@ def combine_usafacts_and_jhu(signal, geo, date_range, fetcher=covidcast.signal):
         combined_df = usafacts_df
     combined_df.rename(COLUMN_MAPPING, axis=1, inplace=True)
 
-    if geo in ["hhs", "nation"]:
```

Comment:
Side comment: selecting only megafips 72000 from JHU (line 114) seems like it would get deaths which are only reported at the mega level, but it's going to miss all the case information which is stored in 72XXX codes. Are we reporting PR case information?

Reply:
Hmmm that's a good question. In fact, I dont even see 72000 in  this file: https://raw.githubusercontent.com/CSSEGISandData/COVID-19/master/csse_covid_19_data/csse_covid_19_time_series/time_series_covid19_confirmed_US.csv, only individual PR fips. Maybe they changed? cc @krivard.

If this needs to be fixed lets open an issue and do it separately.

Reply:
JHU reporting of PR has changed several times now. Agree we should fix it separately.

Reply:
#792 opened

Reply:
Oh, the 72000 is our code. They use 63072999 for the Unassigned category.

> We are now providing the confirmed cases for Puerto Rico at the [county] level. Confirmed cases before 5/6 are categorized into Unassigned, Puerto Rico [63072999] in `time_series_covid19_confirmed_US.csv`. Meanwhile, deaths are all grouped into Unassigned, Puerto Rico in `time_series_covid19_deaths_US.csv`